tests/helper.py

```diff
@@ -0,0 +1,52 @@
+from resin import Resin
+from resin import exceptions as resin_exceptions
+from resin import Token
+
+
+class TestHelper(object):
+
+    credential = {
+        'email': 'pythonsdktest@gmail.com',
+        'password': 'resintest',
```

Comment:
@pimterry Do we really want to be storing resin account credentials in plaintext? How are other API / SDK tests done?
(I guess anybody could log in with the details above, change the account's password, and then suddenly all the PythonSDK CI tests would break)

Reply:
Indeed not. The SDK credentials read from the environment, loading a `.env` file automatically if present, so you just create a `.env` with credentials to run tests locally, and we set the env vars in the Travis/etc build config. We should do that here too (and reset these login details), both for security, and because that way you can use different login details locally/in CI. If you use the same login details everywhere then running the tests locally will break any running CI builds.

Reply:
Fixed. I've created another PR just to see the test result https://github.com/resin-io/resin-sdk-python/pull/69 since travis doesn't allow PR from downstream repo to access encrypted variables.